core/tx_list.go

```diff
@@ -344,18 +344,18 @@ func (l *txList) Add(tx *types.Transaction, priceBump uint64) (bool, *types.Tran
 		}
 		// thresholdFeeCap = oldFC  * (100 + priceBump) / 100
 		a := big.NewInt(100 + int64(priceBump))
-		aFeeCap := new(big.Int).Mul(a, oldGasFeeCap)
-		aTip := a.Mul(a, oldGasTipCap)
+		thresholdFeeCap := new(big.Int).Mul(a, oldGasFeeCap)
+		thresholdTip := a.Mul(a, oldGasTipCap)
 
 		// thresholdTip    = oldTip * (100 + priceBump) / 100
 		b := big.NewInt(100)
-		thresholdFeeCap := aFeeCap.Div(aFeeCap, b)
-		thresholdTip := aTip.Div(aTip, b)
+		newFeeCap := new(big.Int).Mul(b, newGasFeeCap)
+		newTip := a.Mul(b, newGasTipCap)
 
 		// Have to ensure that either the new fee cap or tip is higher than the
 		// old ones as well as checking the percentage threshold to ensure that
 		// this is accurate for low (Wei-level) gas price replacements
-		if newGasFeeCap.Cmp(thresholdFeeCap) < 0 || newGasTipCap.Cmp(thresholdTip) < 0 {
+		if newFeeCap.Cmp(thresholdFeeCap) < 0 || newTip.Cmp(thresholdTip) < 0 {
```

Comment:
This comment is hard for me to read. Does it say that *either* has to be higher by the required percentage? Because the code looks like *both* have to be increased by `priceBump` percent.